src/main/java/org/jabref/gui/preferences/preview/PreviewTabViewModel.java

```diff
@@ -153,13 +163,13 @@ public void setPreviewLayout(PreviewLayout selectedLayout) {
                     Localization.lang("Parsing error") + ": " + Localization.lang("illegal backslash expression"), exception);
         }
 
-        if (selectedLayout instanceof TextBasedPreviewLayout layout) {
-            sourceTextProperty.setValue(layout.getText().replace("__NEWLINE__", "\n"));
```

Comment:
No more replace?

Reply:
forget it, i forgot we put it in the textbasedlayout.

Reply:
moved to getText in The class itself